src/styles/_colors.scss

```
@@ -1,2 +1,3 @@
 $primary: #b88e2f;
 $secondary: #fff3e3;
+$productBG: #f5f5f6;
```

```suggestion
$product-background: #f5f5f6;
```
Dont abreviate if posible (thats for boomers 👨‍🦳) that would be better for readability and also looks.

Also for css naming we use always `kebab-case`